src/components/Users/CreateUserForm.tsx

```diff
@@ -180,7 +196,9 @@ export default function CreateUserForm({ onSubmitSuccess }: Props) {
             name="first_name"
             render={({ field }) => (
               <FormItem>
-                <FormLabel>First Name</FormLabel>
+                <FormLabel>
+                  First Name<span className="text-red-500">*</span>
+                </FormLabel>
```

Comment:
```suggestion
                <FormLabel required>
                 First Name
</FormLabel>
```
Same for those below, add translations as well.